src/vue/components/grammar.vue

```diff
@@ -23,44 +60,162 @@ export default {
   mixins: [DependencyCheck],
   components: {
     backIcon: BackIcon,
-    alphTooltip: Tooltip
+    alphTooltip: Tooltip,
+    progressBar: ProgressBar
+  },
+  data () {
+    return {
+      centralLanguageCode: null,
+      currentLanguageCode: null,
+      currentUrl: null,
+      waitingForGrammar: false,
+      languageList: {
+        lat: {
+          languageID: Constants.LANG_LATIN,
+          languageCode: 'lat',
+          title: 'Latin Grammar',
```

Comment:
since we're hardcoding the titles anyway right now, let's put the grammar names in. (Sorry I missed that this is where that was set in the prior review). I.e. for Latin let's use "New Latin Grammar (Bennett)" and for Greek "A Greek Grammar for Colleges (Smyth)".

Reply:
Fixed